plugins/ci/pkg/ci/git.go

```diff
@@ -87,9 +89,30 @@ func getGitInfo(cmdExecutor cmdInDirExecutor, ciRunner models.CIRunnerVal, baseR
 			logrus.Warnf("Unable to get GIT branch name: %v", err)
 			gitCommandFail = true
 		}
+		refHash := os.Getenv("MASTER_HASH")
```

Comment:
Can we resuse `getGitInfo` here?

https://github.com/FairwindsOps/insights-plugins/blob/4db11ca5df2c1e80794cd7d15178058dfd764c76/plugins/ci/pkg/ci/git.go#L30

Reply:
Maybe we should add `modified files` to `getGitInfo`

Reply:
Can't, I tried to reuse it but it gives me different result than I was expecting

Reply:
hmm if it gives a different result, then it's going to be inconsistent with how we're doing scans

Reply:
Reverted my new code in favor of the current one